server/app/views/FileUploadViewStrategy.java

```diff
@@ -104,7 +104,7 @@ protected abstract ImmutableList<InputTag> fileUploadFields(
   /**
    * Method to render the UI for uploading a file.
    *
-   * @param params the information needed to render a file upload view
+   * @param params the information needed to rener a file upload view
```

Comment:
**nitpick:** typo

Reply:
Should be from an old commit - I accidentally modified this file :)